src/Carousel.vue

```diff
@@ -1,48 +1,40 @@
 <template>
-  <div
-    class="VueCarousel"
-    v-bind:class="{ 'VueCarousel--reverse': paginationPosition === 'top' }"
-  >
+  <transition name="fade">
```

Comment:
In fact, this transition 'do' the job. And I'm not 100% sure is it only my other abstraction for support SSR or something like this can exist. 

As I was watched results in my nuxt demo error with a message that some block is different on client-side and server-side disappear. 

IMO it looks just like my abstraction to deal with SSR and it's fake but without that, I have errors related to ssrNode's and ssrStyle's errors in the console. 

I recommend reproducing this problem on your own with nuxt and this bundle